CONTRIBUTING.md

```diff
@@ -2,483 +2,20 @@
-use that for the build:
-
-Setup some chroots for sbuild with this script
-```shell
-bash ./tools/setup_sbuild.sh
-```
-
-```shell
-debuild -S
-sbuild --dist=<target> ../ubuntu-advantage-tools_*.dsc
-# emulating different architectures in sbuild-launchpad-chroot
-sbuild-launchpad-chroot create --architecture="riscv64" "--name=focal-riscv64" "--series=focal
-```
-
-> Note: Every so often, it is recommended to update your chroots.
-> ```bash
-> # to update a single chroot
-> sudo sbuild-launchpad-chroot update -n ua-xenial-amd64
-> # this script can be used to update all chroots
-> sudo PATTERN=\* sh /usr/share/doc/sbuild/examples/sbuild-debian-developer-setup-update-all
-> ```
-
-### Setting up an lxc development container
-```shell
-lxc launch ubuntu-daily:xenial dev-x -c user.user-data="$(cat tools/ua-dev-cloud-config.yaml)"
-lxc exec dev-x bash
-```
-
-### Setting up a kvm development environment with multipass
-**Note:** There is a sample procedure documented in tools/multipass.md as well.
-```shell
-multipass launch daily:focal -n dev-f --cloud-init tools/ua-dev-cloud-config.yaml
-multipass connect dev-f
-```
-
-## Code Formatting
-
-The `ubuntu-advantage-client` code base is formatted using
-[black](https://github.com/psf/black), and imports are sorted with
-[isort](https://github.com/PyCQA/isort).  When making changes, you
-should ensure that your code is blackened and isorted, or it will
-be rejected by CI.
-Formatting the whole codebase is as simple as running:
-
-```shell
-black uaclient/
-isort uaclient/
-```
-
-To make it easier to avoid committing incorrectly formatted code, this
-repo includes configuration for [pre-commit](https://pre-commit.com/)
-which will stop you from committing any code that isn't blackened.  To
-install the project's pre-commit hook, install `pre-commit` and run:
-
-```shell
-pre-commit install
-```
-
-(To install `black` and `pre-commit` at the appropriate versions for
-the project, you should install them via `dev-requirements.txt`.)
-
-## Daily Builds
-
-On Launchpad, there is a [daily build recipe](https://code.launchpad.net/~canonical-server/+recipe/ua-client-daily),
-which will build the client and place it in the [ua-client-daily PPA](https://code.launchpad.net/~ua-client/+archive/ubuntu/daily).
+### Reference
 
-## Releasing ubuntu-advantage-tools
-See [How to release a new version of UA](./contributing-docs/howtoguides/how_to_release_a_new_version_of_ua.md)
+* [Terminology](./dev-docs/references/terminology.md)
+* [Architecture](./dev-docs/references/architecture.md)
+* [What happens during attach](./dev-docs/references/what_happens_during_attach.md)
+* [Enabling a service](./dev-docs/references/enabling_a_service.md)
+* [Directory layout](./dev-docs/references/directory_layout.md)
+* [Testing](./dev-docs/references/testing.md)
+* [Building](./dev-docs/references/building.md)
+* [Code Formatting](./dev-docs/references/code_formatting.md)
+* [Daily Builds](./dev-docs/references/daily_builds.md)
```

Comment:
I agree with the `how to guide` part, but I think the other sections are still `references`, since we are mainly describing the internals of UA, not actually discussing or explaining any level concept there. But let me know if you still disagree

Reply:
Thanks @lucasmoura I re-read some of diataxis.fr and now I agree with you here on the references